doc/api/module.md

```diff
@@ -931,6 +931,46 @@ added:
 `path` is the resolved path for the file for which a corresponding source map
 should be fetched.
 
+### `module.prepareStackTrace(error, trace)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `error` {Object}
+* `trace` {Object\[]}
+* Returns: {string} Returns formatted stack trace.
+
+This is the default implementation Node.js stack trace formatter. When source
+map support is enabled, this function will attempt to map the stack trace to
+the original source locations. This can be used to override the default stack
+trace formatter.
+
+It is discouraged to modify the stack trace as it can lead to confusion when
+debugging with the stack traces.
+
+```mjs
+import { prepareStackTrace } from 'node:module';
+Error.prepareStackTrace = (error, trace) => {
+  // Filter out custom entries.
+  trace = trace.filter((it) => {
+    return !it.getFileName().includes('my-internal-module');
+  });
+  return prepareStackTrace(error, trace);
+};
```

Comment:
I think we should be cautious about documenting the API here, because the stability of these arguments are not guaranteed by us. A link to https://v8.dev/docs/stack-trace-api should be enough, and we can just put it down as

```suggestion
import { prepareStackTrace: defaultPrepareStackTrace } from 'node:module';
Error.prepareStackTrace = (error, trace) => {
  // Modify error and trace.
  return defaultPrepareStackTrace(error, trace);
};
```

(Or maybe just export this as `defaultPrepareStackTrace`)